lib/dinos.js

```diff
@@ -210,15 +210,29 @@ function _compareDinos(a, b) {
   return 0;
 }
 
+function getStaffField(fullProfile, field) {
+  try {
+    if (
+      field in fullProfile &&
+      fullProfile[field].metadata.visibility !== "private"
+    ) {
+      return fullProfile[field].value;
```

Comment:
Is there any chance that there will be a `values` attribute also?

Reply:
Not in there. But good catch.